airflow/providers/amazon/aws/operators/redshift_sql.py

```diff
@@ -43,12 +43,14 @@ class RedshiftSQLOperator(SQLExecuteQueryOperator):
     template_fields: Sequence[str] = (
         "sql",
         "redshift_conn_id",
+        "conn_id",
```

Comment:
Adding conn_id is good but also remove redshift_conn_id

Reply:
ah ok that makes sense. Thanks for the help!